AlinaFirsova/Homework 1/Homework_1_Firsova.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <title>Cubes</title>
+<style type="text/css">
+ @import"style.css"
+  </style>
+ </head>
+<body background="img\back.png">

```

Comment:
@AlinaFirsova Comment related to the whole doc: please move all inline styles to separate css file. You can set classes to tags and use them in css file to set appropriate properties